### ISSUE TITLE
nothing happens

### ISSUE THREAD

**Issue by gaemil** [0x1iii1ii/PPPwn-Luckfox#109]
Sorry if I ask a previously answered question, but I couldn't find anything similar.
After successfully installing pppwn_luckfox_v1.2.5_pro_max_image_FW_11.00 on Luckfox and connecting the board to PS 4, nothing happens.
I saw that autorun is disabled for web versions, but I don't use a web server, and autorun should be enabled by default.
Please tell me what needs to be fixed?

P.S. Previously, there was a previous version of pppwn for luckfox and it worked without problems, I decided to update to the latest version, now nothing works

**Comment by 0x1iii1ii** [comment]
> Make sure to set the PPPoE username/password to ppp for the feature below to work and for the web to run properly

**Comment by sourceblaak** [comment]
> Sorry if I ask a previously answered question, but I couldn't find anything similar. After successfully installing pppwn_luckfox_v1.2.5_pro_max_image_FW_11.00 on Luckfox and connecting the board to PS 4, nothing happens. I saw that autorun is disabled for web versions, but I don't use a web server, and autorun should be enabled by default. Please tell me what needs to be fixed?
> 
> P.S. Previously, there was a previous version of pppwn for luckfox and it worked without problems, I decided to update to the latest version, now nothing works

I agree. I had a perfectly running jailbreak and luckfox, I could plug it into my router and ssh into the device if I needed. Now it just doesn't work. Period. Connection times out every time. Whether I plug it into the router or the PS4. Everything was followed to T.  I've done it before successfully so it's not a user issue.

**Comment by 0x1iii1ii** [comment]
if some version works best for you, you don't need to upgrade, just stick to it!
